src/preset_cli/cli/superset/sync/dbt/command.py

```diff
@@ -160,17 +157,18 @@ def dbt_core(  # pylint: disable=too-many-arguments, too-many-branches, too-many
             "The managed externally feature was only introduced in Superset v1.5."
             "Make sure you are running a compatible version."
         )
-        log_warning(warn_message, UserWarning)
+        _logger.debug(warn_message)
     if file_path.name == "manifest.json":
-        deprecation_notice = True
-        warn_message = (
-            "Passing the manifest.json file is deprecated. "
-            "Please pass the dbt_project.yml file instead."
-        )
-        log_warning(warn_message, DeprecationWarning)
         manifest = file_path
         profile = project = project or "default"
     elif file_path.name == "dbt_project.yml":
+        deprecation_notice = True
+        warn_message = (
+            "Passing the dbt_project.yml file is deprecated and "
+            "will be removed in a future version. "
+            "Please pass the manifest.json file instead."
```

Comment:
lol

Reply:
Hoping to have a single supported method soon.